src/main/kotlin/net/atos/zac/app/besluit/BesluitPublicationDateMissingException.kt

```diff
@@ -0,0 +1,8 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.app.besluit
+
+class BesluitPublicationDateMissingException(message: String) : BesluitException(message)
```

Comment:
I started translating `besluit` to `decision` at some point. Maybe we should make the decision (sorry..) to use decision instead of besluit? I think we did make this decision to use English in most cases, especially for new code? Of course besluit is a term in the ZGW APIs which makes this a bit tricky maybe. But then so is `publicatieDatum`.. 

But probably better left for another PR I would say.

Reply:
Yep. Will do that renaming in a follow up PR